packages/button/button.scss

```diff
@@ -56,9 +56,11 @@ $button-primary-background-color--inverted: $hvit;
 .jkl-button-wrapper {
     height: $button-height--normal;
     overflow: hidden;
-    @supports (clip-path: inset(-5% -5% -5% -5%)) {
+    padding: 7px;
```

Comment:
Tror vi skal legge den i en "variabel" og gi den en liten forklaring på magien :)

Reply:
Hva ville du kalt den variabelen? jkl-button-wrapper-padding ?
Og forklaring sånn: // This padding ensures that there is enough space around the button so that it is fully visible during hover and focus inside the wrapper. Note that this padding is only necessary for browsers that doesn't support clip-path.

Reply:
`$button-wrapper-animation-safe-area` kanskje? Kommentern ser bra ut